src/index.ts

```diff
@@ -316,7 +324,7 @@ let lastWasConstant = false;
 export function isConstant(
   src: string,
   constants: any = EMPTY_OBJECT,
-  options: BabylonOptions = EMPTY_OBJECT,
+  options: ParserOptions = EMPTY_OBJECT,
 ) {
```

Comment:
How about, after parsing the expression:

```ts
if (!b.isExpression(ast)) {
  return (lastWasConstant = false);
}
```

Then we can continue to use the less weird `b.Expression` for the type of `expressionToConstant`.

Reply:
Actually, I think this can be resolved just by bumping the version for @babel/parser

Reply:
does not seem to resolve the error https://gist.github.com/zackschuster/864c2cafda2608f816779bf4d5df8a20